apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.ts

```diff
@@ -116,6 +117,7 @@ router.post(
       const origHash = req.body.orig_hash;
       questionInfo.title = req.body.title;
       questionInfo.topic = req.body.topic;
+      questionInfo.tags = req.body.tags;
```

Comment:
This breaks if there's just a single tag selected.

Reply:
Updated in [d426d26](https://github.com/PrairieLearn/PrairieLearn/pull/10973/commits/d426d26b01edba612257c99691275d279950c468) to handle if this is not an array.